templates/article-page/article-page.js

```diff
@@ -5,21 +5,20 @@ import {
 function createTemplateBlock(main, blockName, gridName) {
   const gridNameValue = gridName || blockName;
   const section = document.createElement('div');
-  section.className = `article-template-autoblock article-template-grid-${gridNameValue}`;
+  section.className = `section article-template-autoblock article-template-grid-${gridNameValue}`;
```

Comment:
If you use:
```suggestion
  section.classList.add('article-template-autoblock', `article-template-grid-${gridNameValue}`);
```
you don't have to add the default class again since you are just appending to the list